src/main/java/sora/Parser.java

```diff
@@ -0,0 +1,47 @@
+package sora;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * Parser parses the User's input.
+ * It deals with making sense of the user command.
+ * It identifies mainCommand and additional details in the User's input.
+ */
+public class Parser {
+    /**
+     * Returns an ArrayList that parses the User's command.
+     * Identifies mainCommand and additional details in the User's input.
+     *
+     * @param command User's command / input.
+     * @return ArrayList containing parsed command.
+     */
+    public static ArrayList<String> parse(String command) {
+        ArrayList<String> parsedResult = new ArrayList<>();
+        String[] parse_1 = command.split(" ", 2);
```

Comment:
Perhaps this could be better named? Maybe something like parseTemp or parsedArr instead of using _

Reply:
Nice Catch. Will Rename.